### DISCUSSION TITLE
"What's new" when an update is triggered

### DISCUSSION THREAD

**Discussion by zacjones93** [johnlindquist/kit#275] [Q&A] [answered]
Where is the best place to look to see what's new with Script Kit when an update is triggered?

Just saw an update go through and wanted to know if there is a specific place this occurs. I'll poke around!

**Comment by johnlindquist** [comment, marked as answer]
I making videos about what's new today. I'll post them in the announcements section 👍

**Reply by zacjones93** [reply]
Woo,thanks! #282
